libraries/AP_Filesystem/AP_Filesystem.cpp

```diff
@@ -61,6 +61,7 @@ const AP_Filesystem::Backend AP_Filesystem::backends[] = {
     { nullptr, fs_local },
 #ifdef HAL_HAVE_AP_ROMFS_EMBEDDED_H
     { "@ROMFS/", fs_romfs },
+    { "@ROMFS", fs_romfs },
```

Comment:
it would be nicer to fix it so we don't need to list it twice. Just list it as @ROMFS and cope with the '/'
we should fix @SYS in the same way too